app/src/main/res/values-zh/strings.xml

```diff
@@ -0,0 +1,96 @@
+<resources>
```

Comment:
I'm not accepting translations for gotify, see this thread https://github.com/gotify/android/issues/176